development/ceph-base-focal-quincy/bundle.yaml

```diff
@@ -0,0 +1,41 @@
+name: ceph-base
+machines:
+  '0':
+    series: focal
+  '1':
+    series: focal
+  '2':
+    series: focal
+relations:
+- - ceph-osd:mon
+  - ceph-mon:osd
+series: focal
+applications:
+  ceph-mon:
+    annotations:
+      gui-x: '750'
+      gui-y: '500'
+    charm: ch:ceph-mon
+    channel: quincy/edge
+    num_units: 3
+    options:
+      expected-osd-count: 3
+      source: distro
+    to:
+    - lxd:0
+    - lxd:1
+    - lxd:2
+  ceph-osd:
+    annotations:
+      gui-x: '1000'
+      gui-y: '500'
+    charm: ch:ceph-osd
+    channel: quincy/edge
+    num_units: 3
+    options:
+      osd-devices: /dev/sdb
+      source: distro
```

Comment:
focal-quincy should be configuring focal-yoga UCA.

Reply:
ack